disk_store.go

```diff
@@ -184,7 +183,7 @@ func (d *DiskStore) Delete(key string) error {
 
 	// mark as tombstone
 	h.MarkTombStone()
-	r := Record{Header: h, Key: key, Value: "", RecordSize: headerSize + h.KeySize + h.ValueSize}
+	r := Record{Header: h, Key: key, Value: value, RecordSize: headerSize + h.KeySize + h.ValueSize}
```

Comment:
hey, did this cause a regression? Why did we change

Reply:
No any regression. Just used the var defined on line 181: 
`value := ""`